integtest/spec/helper/book.rb

```diff
@@ -24,6 +24,10 @@ class Book
   # Should this book allow overriding :edit_url:? Defaults to false.
   attr_accessor :respect_edit_url_overrides
 
+  ##
+  # The language of the book. Defaults to `en`.
```

Comment:
There's no functionality to set/override the `@lang`, is that intentional?
This comment says the lang default is 'en' but you can't set it otherwise...

Reply:
Ah! I certainly meant to....

Reply:
I set it with `book.lang` [here](https://github.com/elastic/docs/pull/1191/files#diff-88db19d5cf245e6c1d6566dc005be3abR427).  It seems to work...